devops/list-all-repos-in-org/list-all-repos-in-org.sh

```diff
@@ -0,0 +1,13 @@
+# !/bin/sh
+
+organization="https://dev.azure.com/mseng"
+
+projects="$(az devops project list --org "${organization}" -o tsv --query [].name)"
```

Comment:
Requires the devops extension as a prereq. Make sure this is mentioned in a comment after the bang.

Reply:
added a comment, please take a look

Reply:
@sptramer : once this PR goes through, any idea where this script will be visible?
I mean are these scripts hosted/ exposed in some other portals/ sites as well?

Reply:
Only available on GitHub or with a specific markdown extension.